src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -582,20 +582,10 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
             });
         }
 
-        // workaround for imprecise timestamp that was including the cursor itself
-        const addMillisecond = (cursor: string) =>
-            format(
-                addMilliseconds(parseISO(cursor), 1),
-                "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'",
-            );
         if (cursor) {
-            query = query.where(
-                'features.created_at',
-                '>',
-                addMillisecond(cursor),
-            );
+            query = query.where('features.created_at', '>=', cursor);
```

Comment:
in this version we use cursor from the next feature that was not part of the previous list